code/modules/antagonists/blob/blob_abilities.dm

```diff
@@ -1119,6 +1125,12 @@
 			repeatable--
 		if (repeatable == 0)
 			owner.available_upgrades -= src
+		if (prob(80))
+			owner.playsound_local(owner.loc, 'sound/voice/blob/blobup1.ogg', 50, 1)
+		else if (prob(50))
+			owner.playsound_local(owner.loc, 'sound/voice/blob/blobup2.ogg', 50, 1)
+		else
+			owner.playsound_local(owner.loc, 'sound/voice/blob/blobup3.ogg', 50, 1)
```

Comment:
single quotes

Reply:
There is specific meanings but these are used by several different sounds, this is the cleanest way of doing it.

Reply:
wrong comment

Reply:
also Big Doubt on 'cleanest way of doing it'